appveyor.yml

```diff
@@ -19,6 +19,7 @@
     - src\packages -> src\**\packages.config
 
   before_build:
+    - appveyor DownloadFile https://dist.nuget.org/win-x86-commandline/v3.2.0/nuget.exe
```

Comment:
yes, we are still facing some nuget bugs, they literally depends on the configuration of the VM we run on, in this case we're forcing a specific nuget version that works as expected.